extensions/pom.xml

```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>org.mule</groupId>
             <artifactId>mule-core-tests</artifactId>
-            <version>${project.version}</version>
+            <version>${mule.version}</version>
```

Comment:
to solve that problem we're going to do a separate pom for the extensions that are not shipped with mule. For the ones that are part of the distributions, you should refer to by project.version. This PR is invalid